app/src/main/java/me/ccrama/redditslide/SettingValues.java

```diff
@@ -513,4 +514,12 @@ public static boolean hasSort(String subreddit) {
     public enum ColorMatchingMode {
         ALWAYS_MATCH, MATCH_EXTERNALLY
     }
+
+    public enum NightModeState {
+        DISABLED, MANUAL, AUTOMATIC;
```

Comment:
If these get moved around or get out of sync with the resource array, everything breaks. Notably.